unit/Makefile

```diff
@@ -1,19 +1,14 @@
-SRC = cpp_parser.cpp \
-      cpp_scanner.cpp \
-      elf_reader.cpp \
-      float_utils.cpp \
-      ieee_float.cpp \
-      json.cpp \
+CATCH_TESTS = catch_example.cpp \
+              util/sharing_map.cpp \
+              #Empty last line
+
+SRC = catch_entry_point.cpp \
       miniBDD.cpp \
-      osx_fat_reader.cpp \
```

Comment:
Dangling files aren't great. (Yes, effectively the entire directory had been dangling.) Any chance to get those fixed instead?